test/e2e/framework/util.go

```diff
@@ -359,7 +359,7 @@ func SkipIfMultizone(c clientset.Interface) {
 		Skipf("Error listing cluster zones")
 	}
 	if zones.Len() > 1 {
-		Skipf("Requires more than one zone")
```

Comment:
should it be `at most`? given the if condition

Reply:
Indeed, in the case there is no zone at all (i.e. no cloud provider integration I think) `zones` may be empty. I'm going to fix that.